CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Introduction
+
+Thank you for your interest in contributing to Houdini! This guide aims to facilitate contribution to our project. Follow all the steps below to submit your contribution:
+
+1. Fork the Repository
+2. Comment "**WIP**" on the existing Issue or create an issue according to the implemented issue templates.
```

Comment:
**WIP** wouldn't be helpful because it doesn't connect the issue with the PR related to the issue and doesn't give the rest of us any information. I recommend considering how this interacts with draft PRs as well as the ability to associate  PR with a related issue.

Reply:
okay, I already changed the step by step for the contribution in houdini